memory-viz/src/memory_model.ts

```diff
@@ -5,10 +5,21 @@ import merge from "deepmerge";
 import { collections, immutable, presets, setStyleSheet } from "./style";
 import { config } from "./config";
 import { DOMImplementation, XMLSerializer } from "@xmldom/xmldom";
-import { DrawnEntity } from "./types";
+import {
+    DrawnEntity,
+    VisualizationConfig,
+    Rect,
+    Primitive,
+    Style,
+} from "./types";
+import { isArrayOfType } from "./typeguards";
+import { RoughSVG } from "roughjs/bin/svg";
+import { Config, Options } from "roughjs/bin/core";
+import type * as fsType from "fs";
+import type * as CSS from "csstype";
```

Comment:
I imagine this package has to be added to the devDependencies, otherwise how does this import work?